app/controllers/workouts_controller.rb

```diff
@@ -59,15 +48,6 @@ def update
     end
   end
 
-  def mark_finished
```

Comment:
Method removed, but did you remove the corresponding method calls? (cf workouts/show view)
Also if you're sure we don't need this anymore you can remove the route and the policy